docs/reference/commands/new.rst

```diff
@@ -21,7 +21,12 @@ The following options can be provided at the command line.
 ---------------------------------------------
 
 A local directory path or URL to use as a cookiecutter template for the new
-project. Briefcase will attempt to use a template branch matching the version
+project.
+
+``--template-branch <branch>``
+-----------------------------
+The branch of the cookiecutter template repo to use for the new project.
```

Comment:
Minor stylistic nit - there should be a blank line between the `---` and the first line of content in the paragraph; and the paragraph should be word-wrapped to 80 characters.